mysql/assets/monitors/mysql_replication_seconds_behind_master.json

```diff
@@ -21,5 +21,8 @@
         "thresholds": {
             "critical": 3000
         }
-    }
+    },
+	"recommended_monitor_metadata": {
```

Comment:
Why are we adding this?

Reply:
Fixing payload to be consistent, updating title 😛